docs/docs/theme-ui.md

```diff
@@ -0,0 +1,264 @@
+export default {
+  colors: {
+    text: "#333",
+    background: "#fff",
+    primary: "#639",
+    secondary: "#ff6347",
+  },
+  // highlight-start
+  fonts: {
+    body: "system-ui, sans-serif",
+    heading: "system-ui, sans-serif",
+    monospace: "Menlo, monospace",
+  },
+  fontWeights: {
+    body: 400,
+    heading: 700,
+    bold: 700,
+  },
+  lineHeights: {
+    body: 1.5,
+    heading: 1.125,
+  },
+  fontSizes: [12, 14, 16, 20, 24, 32, 48, 64, 72],
+  // highlight-end
+}
+```
+
+Next add values for use in margin and padding.
+
+```js:title=src/gatsby-plugin-theme-ui/index.js
+export default {
+  colors: {
+    text: "#333",
+    background: "#fff",
+    primary: "#639",
+    secondary: "#ff6347",
+  },
+  fonts: {
+    body: "system-ui, sans-serif",
+    heading: "system-ui, sans-serif",
+    monospace: "Menlo, monospace",
+  },
+  fontWeights: {
+    body: 400,
+    heading: 700,
+    bold: 700,
+  },
+  lineHeights: {
+    body: 1.5,
+    heading: 1.125,
+  },
+  fontSizes: [12, 14, 16, 20, 24, 32, 48, 64, 72],
+  // highlight-start
+  space: [0, 4, 8, 16, 32, 64, 128, 256, 512],
+  // highlight-end
+}
+```
+
+Feel free to include as many additional values as you'd like in your theme.
+Read more about theming in the [Theme UI documentation](https://theme-ui.com/theming).
+
+## Adding styles to elements
+
+Theme UI uses a custom [JSX pragma][] comment to add support for Emotion's `css` prop,
+which can be used to style elements.
+By default JSX compiles to the `React.createElement` function.
+When you add the `/** @jsx jsx */` comment to the top of a file, JSX will compile to the function name in the comment instead.
+For example, `<button />` would compile to `jsx('button')`.
+Theme UI's `jsx` function still renders React elements but lets you use the `css` prop for styling.
+
+[jsx pragma]: https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#pragma
+
+The following example uses the `css` prop to add styles to an element.
+
+```js:title=src/components/header.js
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+
+export default props => (
+  <header
+    css={{
+      padding: 32,
+      color: "#fff",
+      backgroundColor: "#639",
+    }}
+  >
+    {props.children}
+  </header>
+)
+```
+
+With Emotion's API, values from the theming context can be referenced by passing a function to the `css` prop.
+
+```js:title=src/components/header.js
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+
+export default props => (
+  <header
+    css={theme => ({
+      padding: theme.space[4],
+      color: theme.colors.background,
+      backgroundColor: theme.colors.primary,
+    })}
+  >
+    {props.children}
+  </header>
+)
+```
+
+Theme UI includes the [`sx` prop][], which can be used as a replacement for the `css` prop.
+The `sx` prop allows values from the theming context to be referenced with a terser syntax.
+
+[`sx` prop]: https://theme-ui.com/sx-prop
+
+The example from above can be rewritten like this:
```

Comment:
I'm hesitant to introduce the shorthand into these docs, because it's _so much_ to learn at once and we want this to  be as easy as possible. Maybe making it clear that this is completely optional would help, with an "Advanced usage" heading or something. The way it's worded kinda makes it sound like everyone would want to rewrite it that way, when it could introduce more cognitive overhead for people following along (even with the "can" part).

Reply:
In the [Theme UI docs](https://theme-ui.com/sx-prop) it introduces the `sx` prop first, and doesn't mention how to use the `css` prop until the end. The `sx` prop is the preferred way to add styles (i.e. without the `sx` prop, Theme UI isn't doing much beyond what you can do with Emotion), and `css` is sort of an escape hatch. I moved the `css` prop section before in an attempt to explain how it works for people who are familiar with Emotion, but maybe that could be left out of this doc entirely. Curious to hear what others think

Reply:
I think if `sx` is the preferred pattern we want to promote, then it should be prioritized in this doc.